docs/cli.md

```diff
@@ -93,6 +94,28 @@ In the case where your application resides in a directory other than your reposi
 folder, you can specify that directory using the `--context-dir` parameter. The
 specified directory will be used as your application root folder.
 
+#### Injecting directories to build
+
+In case you want to inject a files that should be available only during the
+build time (iow. when the `assemble` script is invoked), you can specify the
+directories from where the files will be copied into the container that runs
+this script. To do so you can invoke S2I as following:
+
+```console
+$ s2i build --inject /mydir:/container/dir file://source builder-image output-image
+```
+
+In this case the content of the `/mydir` directory will get copied into
+`/container/dir` inside the container which runs the assemble script.
+After the `assemble` script finish, all files copied will be truncated and thus
+not available in the output image.
+
+You can also specify multiple directories which in that case must be delimited
+by `,`. For example: `--inject /dir1:/container/dir1,/dir2/container/dir2,...`.
```

Comment:
I think because I watched Dope last night I got this thing in my head: slippery slope...
Adding custom format for this one flag opens an unnecessary precedent to have inconsistent CLI.
I'd much rather leave handling of flags to cobra.